x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/search/action/AsyncStatusResponse.java

```diff
@@ -36,6 +36,12 @@ public class AsyncStatusResponse extends ActionResponse implements SearchStatusR
     private final int failedShards;
     private final RestStatus completionStatus;
 
+    // optionally include a _clusters object in the status response. This is used for CCS.
+    // This only needs to be handled on the coordinator, so this instance is transient (not Writeable).
+    // The SearchResponse has a Writeable copy that gets written to the async-search
+    // system index, so we can retrieve it from there after the task is finished
+    private transient SearchResponse.Clusters clusters;
```

Comment:
what happens if node1 is the coordinator for the ongoing search, and you ask node2 for the status of that same async search?

Reply:
Oh, thanks for that. I'm not sure how that works. Would the node 2 coordinator be contacting node 1 to get the in-memory async search status?

So in that case we would need to have Clusters added to the transport serialization so that node 1 can send it to node 2. Is that right?

Reply:
Yeah, I set up a 2-node cluster and tested this case and if you go to node2 for status you don't get the `_clusters` entry, so I'll need to make that field Writeable and do a Transport Version bump in this PR.